app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java

```diff
@@ -0,0 +1,47 @@
+package com.eis.geoCalendar.network.SMS;
+
+import com.eis.geoCalendar.gps.GPSPosition;
+import com.eis.smslibrary.SMSPeer;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class SMSStringEventTest {
+    private static final String TEST_NUMBER = "+390000111222";
+    private static final String TEST_DESCRIPTION = "Name: description";
+    private static final double LAT = 13;
+    private static final double LON = 14;
+
+    private GPSPosition testPosition;
+    private SMSStringEvent testEvent;
+    private SMSNetworkEventUser testUser;
+    private SMSPeer testPeer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'testPeer' could be replaced by a local variable.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395)